### DISCUSSION TITLE
Welcome to Tribler Discussions!

### DISCUSSION THREAD

**Discussion by ichorid** [Tribler/tribler#5525] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Hello, I'm Jay and I'm here because I love technology that empowers it's users in the way that Tribler does. I won't say a lot about myself publicly as I value my privacy which is of course why I use Tribler in the first place, although you're welcome to ask me anything privately. I'm a BitTorrent enthusiast if you will, it's the first and most decentralized free software to exist.

As a Libertarian, it is my end goal to redistribute power and give it back to it's people by way of technology, much like Robin Hood. Power corrupts, and absolute power corrupts absolutely, and the kings and queens of the world have plenty of it. Call me ideological, but I'd say I'm a realist.

I'm very new to the community, in fact I haven't done much at all with the community as of yet, but that's why I'm here. I have a YouTube channel where I'll be posting videos not only on Tribler, but on Bisq, OnionShare and other empowering free software as well. As I mentioned, I'm a BitTorrent enthusiast, so much of what I'm interested in is anything related to BitTorrent or file sharing in general.

**Comment by georgio2080** [comment]
I have read horror stories about people being raided because of tribler. I have not seen any proof on these subjects so I thought I would ask here. From what I read it sounded like people were activating the exit node option. From what I understand the exit node option is now turned off by default? There is no way for the exit node to turn itself on ever?

Does tribler actually block what your isp can see you downloading without anyone having to configure anything? If you were to download something should you use something like tor to copy the magnet link then paste it into tribler? If you do this is there no chance that your isp will send you any kind of copyright claim or no chance that ip trolls will see what you are doing?

Are you required to upload as much as you download? If you upload ratio is lower than your download ratio are you blocked from using tribler or limited on how fast you are able to download files?

Thank you for any info.

**Reply by ghost** [reply]
Use DNS over HTTPS with [SimpleDNSCrypt](https://simplednscrypt.org/) to encrypt all of your DNS queries and your ISP won't see anything.

**Reply by georgio2080** [reply]
Is this something that should be used with tribler or does this work no matter what even if you dont copy the magnet link to your torrent program?

**Reply by ghost** [reply]
Sorry, I wasn't thinking, DoH will only encrypt your DNS queries, so any Tribler activity will not be encrypted with DoH, you should use a VPN tunnel. A VPN tunnel will encrypt all of your Internet traffic, regardless of weather you're using Tribler or not. I personally use [Mullvad](https://mullvad.net/) as my VPN provider.

**Reply by georgio2080** [reply]
Would using tribler to download files and using tor to copy the magnetic link generally be good enough to hide what is being downloaded from ip trolls?

**Reply by ghost** [reply]
Yes, you can use the Tor Browser to get your magnet links.

**Reply by georgio2080** [reply]
By using tor to get magnetic links and using tribler that will protect you from ip trolls and your isp for the most part?

**Reply by devos50** [reply]
> protect you from ip trolls and your isp for the most part?

I would say so. Make sure that your download in Tribler is anonymous though (which is the default option).

**Reply by georgio2080** [reply]
Would just using tor to copy the magnet link into any torrent program be enough to hide the information from ip trolls?

**Comment by devos50** [comment]
> From what I understand the exit node option is now turned off by default? There is no way for the exit node to turn itself on ever?

The exit node option is disabled by default. You can enable it through the settings pane in the anonymity tab, if I remember correctly.

> Does tribler actually block what your isp can see you downloading without anyone having to configure anything?

There are likely a few things that the ISP can see, e.g., the communication in the IPv8 network (which is unencrypted). Note that anonymous downloads are fully encrypted and _should_ not leak data to the ISP. @egbertbouman is more familiar with that technology and can most likely expand on this answer.

> Are you required to upload as much as you download? If you upload ratio is lower than your download ratio are you blocked from using tribler or limited on how fast you are able to download files?

Your 'token balance' is only used when the exit nodes cannot keep up with the demand for data (e.g., during weekends when more people are downloading). In that scenario, the users that have higher token balances get priority and therefore experience more stable and higher anonymous download speeds.

**Comment by egbertbouman** [comment]
> Does tribler actually block what your isp can see you downloading without anyone having to configure anything? 

By default, all the download traffic should be hidden from your ISP. However, there are a few things that aren't tunneled that you should be aware of:
* Search queries and results.
* All channel-related traffic is unencrypted. Also, the magnet links you put in your channel are public.
* BitTorrent health lookups using trackers. We plan to address this in Tribler 7.6 (https://github.com/Tribler/tribler/issues/5262).

> If you were to download something should you use something like tor to copy the magnet link then paste it into tribler?

I'm not sure I understand, but if you get the magnet link from elsewhere using a browser, and you'd like that part to be anonymous too, you'd need to take precautions yourself (e.g. by using TOR). Once the magnet link is copied into Tribler, all download traffic should be tunneled.
